pkg/genericapiserver/serialization_config.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Sorry, should have said this before. but resource_encoding_config seems like a better name than serialization_config since this defines the ResourceEncodingConfig struct.